README.md

```diff
@@ -1 +1,22 @@
-# ros_computer_setup
\ No newline at end of file
+# ros_computer_setup
+Simple script that makes all Clearpath specific changes to a as system that does not support the standard ISO image
+(ie Jeton) to make it operate like our robot standard images.  Makes it work as a standard computer for Husky or Jackal.
+
+## Supported Jetsons
+* TX2 (Kinetic)
+* Nano (Melodic)
+* Xavier AGX (Melodic)
+* Xavier NX (Melodic)
+
+## Usage
+```wget -c https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/install.sh && bash install.sh```
```

Comment:
Is it worth having a curl command instead of wget + bash?  Not really bothered either way, just a question.

Reply:
Only cause `wget` is installed by default.

Reply:
Good point. I forgot curl wasn't included on the jetsons.